### PR TITLE
python3Packages.flametree: enable tests

### PR DESCRIPTION
**Pull request by fabaff**
###### Motivation for this change
Enable tests

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 121736` at 3c24a096 run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>4 packages built successfully:</summary>
  <ul>
    <li>python38Packages.dnachisel</li>
    <li>python38Packages.flametree</li>
    <li>python39Packages.dnachisel</li>
    <li>python39Packages.flametree</li>
  </ul>
</details>

File: pkgs/development/python-modules/flametree/default.nix [hunk 1]
```diff
@@ -1,25 +1,40 @@
...
+    matplotlib
+    pandas
+    pytestCheckHook
+    weasyprint
+  ];
+
+  disabledTests = [
+    # Fontconfig is needed for weasyprint test
+    "test_weasyprint"
+  ];
```
Comment (dotlambda) [inline pkgs/development/python-modules/flametree/default.nix:31]: Does
```suggestion
  preCheck = ''
    export FONTCONFIG_FILE=${fontconfig.out}/etc/fonts/fonts.conf
  ''
```
work?

Comment (fabaff) [reply pkgs/development/python-modules/flametree/default.nix:31]: It does. Unfortunately is there another issue (a file size assertion that fails).

Comment (dotlambda) [reply pkgs/development/python-modules/flametree/default.nix:31]: How about setting `preCheck` anyway and maybe leaving that file size assertion to fix in the future?

**Comment by fabaff** [comment]
Result of `nixpkgs-review pr 121736` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>4 packages built:</summary>
  <ul>
    <li>python38Packages.dnachisel</li>
    <li>python38Packages.flametree</li>
    <li>python39Packages.dnachisel</li>
    <li>python39Packages.flametree</li>
  </ul>
</details>

**Review by prusnak — APPROVED** [review]

**Review by jonringer — APPROVED** [review]
LGTM

Result of nixpkgs-review pr 121736 run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)<details>  <summary>4 packages built:</summary>  <ul>    <li>python38Packages.dnachisel</li>    <li>python38Packages.flametree</li>    <li>python39Packages.dnachisel</li>    <li>python39Packages.flametree</li>  </ul></details>

**Pull request merged**